Source/GitReleaseManager/GitHubProvider.cs

```diff
@@ -366,12 +373,54 @@ private static string ComputeSha256Hash(string asset)
             }
         }
 
+        private async Task AddIssueCommentsAsync(string owner, string repository, string milestone, string tagName)
+        {
+            var issueComment = _configuration.Publish.IssueCommentFormat.ReplaceTemplate(new { owner, repository, Milestone = milestone, tagName });
+            var issues = await GetIssuesFromMilestoneAsync(owner, repository, milestone).ConfigureAwait(false);
+
+            foreach (var issue in issues)
+            {
+                SleepWhenRateIsLimited();
+
+                Logger.WriteInfo(string.Format("Adding published comment for issue #{0}", issue.Number));
+                try
+                {
+                    await _gitHubClient.Issue.Comment.Create(owner, repository, issue.Number, issueComment).ConfigureAwait(false);
```

Comment:
Should we add a check here to make sure that the issue is closed?  Just as a sanity check?

Reply:
Also, can we check to make sure that the new issue comment doesn't already exist on the issue?  Again, just as a sanity check.

Reply:
> Should we add a check here to make sure that the issue is closed? Just as a sanity check?

I don't think that is necessary, as the method returning the issues specifically only requests closed issues. But, sure I can add a check for it to future proof it.

> Also, can we check to make sure that the new issue comment doesn't already exist on the issue? Again, just as a sanity check.

Sure, I can add that.